pubsub/gcloud/aio/pubsub/metrics.py

```diff
@@ -35,6 +35,8 @@
         namespace=_NAMESPACE,
         subsystem=_SUBSYSTEM,
         unit='seconds',
+        buckets=(.01, .1, .25, .5, 1.0, 2.5, 5.0, 7.5, 10.0, 20.0,
```

Comment:
The defaullt buckets seem to be:

``` python
from prometheus_client import Histogram

# Create a Histogram without specifying buckets to see the default ones
histogram = Histogram('request_duration_seconds', 'Description of histogram', unit='seconds')

# Access the default buckets
default_buckets = histogram._upper_bounds
default_buckets
```
Results:
```
[0.005,
 0.01,
 0.025,
 0.05,
 0.075,
 0.1,
 0.25,
 0.5,
 0.75,
 1.0,
 2.5,
 5.0,
 7.5,
 10.0,
 inf]
 ```
 
 Given the 2 plus minutes we're being reported, we might even need to have bigger buckets, say, 60, 90 and 120.

Reply:
I'd be fine going up to 2 minutes as well. I'm not sure the 90 second bucket is necessarily useful. For our purposes, if we're something like 30 seconds behind then we're definitely way too far behind already, I think we just wanted to have a better chance at capturing more accurate upper bounds. Also good to keep in mind this applies to all of our apps, plus the apps of our users who choose to make use of the metrics.